internal/registry/client.go

```diff
@@ -168,7 +168,7 @@ func (c *Client) ModuleVersions(ctx context.Context, module *regsrc.Module) (*re
 
 	for _, mod := range versions.Modules {
 		for _, v := range mod.Versions {
-			log.Printf("[DEBUG] found available version %q for %s", v.Version, mod.Source)
+			log.Printf("[DEBUG] found available version %q for %s", v.Version, module.Module())
```

Comment:
Could we keep mod.Source in the log here?

`found available version %q of %q for %s`, version, source, module

Reply:
What if mod.Source is empty? Should we still print `found available version x.x.x of for provider/module/name`